crates/orchestrator/src/queue/job_queue.rs

```diff
@@ -231,7 +231,8 @@ fn parse_worker_message(message: &Delivery) -> Result<Option<WorkerTriggerMessag
         .borrow_payload()
         .ok_or_else(|| ConsumptionError::Other(OtherError::from("Empty payload".to_string())))?;
     let message_string = String::from_utf8_lossy(payload).to_string().trim_matches('\"').to_string();
-    let trigger_type = WorkerTriggerType::from_str(message_string.as_str()).expect("trigger type unwrapping failed");
+    let trigger_type = WorkerTriggerType::from_str(message_string.as_str())
+        .map_err(|e| ConsumptionError::Other(OtherError::from(e)))?;
```

Comment:
Let's add a `wrap_err` here as well,
seems like a good practice to follow 
WDYT @Mohiiit  ?

Reply:
valid added